saltswap/swapper.py

```diff
@@ -661,13 +651,9 @@ def attempt_identity_swap(self, context, penalty, saltmax=None):
         if mode == "remove salt":
             self.proposal = [ncation, ncation - 1]
             self.cumulative_work = cumulative_work
```

Comment:
Good question. I think recording the proposal and corresponding work is a more efficient way of doing just that. For instance, if the `proposal = [0,1]`, the work would correspond to adding 1salt pair when there is no salt present. Also, if the `proposal = [8,7]`, the work would correspond to removing 1 salt pair when there are 8 present. This will also help with calculating the free energy changes between different states using BAR.